lib/ddtrace/span.rb

```diff
@@ -252,3 +253,6 @@ def now_allocations
     end
   end
 end
+
+# Include extensions after Span (for Ruby 1.9 compatibility)
+Datadog::Span.send(:include, Datadog::Analytics::Span)
```

Comment:
That keeps the `Span` class way more clean, even though we still have an external dependency.